Website/Controllers/WeBlogImportController.cs

```diff
@@ -0,0 +1,140 @@
+﻿using System;
+using System.Web.Http;
+using Sitecore.Jobs;
+using Sitecore.Modules.WeBlog.Import;
+using Sitecore.Services.Core;
+using Sitecore.Services.Infrastructure.Web.Http;
+using Sitecore.StringExtensions;
+
+namespace Sitecore.Modules.WeBlog.Controllers
+{
+    [ServicesController]
+    public class WordPressImportController : ServicesApiController
+    {
+        private string JobHandle { get; set; }
+
+        [HttpPut]
+        [Authorize]
+        public string ImportItems(WordPressImportData data)
+        {
+            var options = new JobOptions("Creating and importing blog", "WeBlog", Context.Site.Name, this, "ImportBlog", new[] { data });
+            var job = JobManager.Start(options);
+            job.Status.Total = 0;
+            JobHandle = job.Handle.ToString();
```

Comment:
Don't use an instance property as it won't be used once the request returns. Instead, just pass the job handle back and use that in subsequent calls (which is already being done). Just don't use a property for this.